bindings/requests.go

```diff
@@ -5,8 +5,20 @@
 
 package bindings
 
-// WriteRequest is the object given to an dapr output binding
-type WriteRequest struct {
-	Data     []byte            `json:"data"`
-	Metadata map[string]string `json:"metadata"`
+// InvokeRequest is the object given to a dapr output binding
+type InvokeRequest struct {
+	Data      []byte            `json:"data"`
+	Metadata  map[string]string `json:"metadata"`
+	Operation string            `json:"operation"`
```

Comment:
OperationType ?

Reply:
Changed

Reply:
I used `OperationKind` as it stutters less